backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/drawings/calibrations.js

```diff
@@ -70,6 +71,11 @@ const validateNewCalibrationData = async (req, res, next) => {
 
 	try {
 		req.body = await schema.validate(req.body);
+
+		const { verticalRange, units } = req.body;
```

Comment:
you could map the rest of the object to an object variable like this:
```suggestion
		const { verticalRange, units, ... others } = req.body;
```

Then you woulnd't need to do the deleteUndefined thing